op-chain-ops/cmd/celo-migrate/main.go

```diff
@@ -78,45 +81,43 @@ var (
 		Usage: "Memory limit in MiB, should be set lower than the available amount of memory in your system to prevent out of memory errors",
 		Value: 7500,
 	}
-	clearAllFlag = &cli.BoolFlag{
-		Name:  "clear-all",
-		Usage: "Use this to start with a fresh new db, deleting all data including ancients. CAUTION: Re-migrating ancients takes time.",
+	clearNonAncientsFlag = &cli.BoolFlag{
```

Comment:
I don't think we need this flag any more, using rsync with the `--delete` flag will make the receiving datadir match the sending datadir exactly. So no need to clear anything.

Reply:
ahh I think I just forgot to delete it, totally agree